src/app/features/tech-record/components/vehicle-technical-record/vehicle-technical-record.component.html

```diff
@@ -62,7 +62,11 @@
       [vehicleTechRecord]="vehicleTechRecord"
     ></app-tech-record-history>
 
-    <app-test-record-summary *appRoleRequired="roles.TestResultView" [testRecords]="(records$ | async) || []" [vehicleTechRecord]="vehicleTechRecord">
+    <app-test-record-summary
+      *appRoleRequired="roles.TestResultView"
+      [testRecords]="(records$ | async) || []"
+      [currentTechRecord]="(currentTechRecord$ | async) || undefined"
```

Comment:
Is the `|| undefined` because `currentTechRecord$` can return `null`?

Reply:
Yes